postgresql-persistence/src/main/java/com/netflix/conductor/aurora/AuroraPriorityLookupDAO.java

```diff
@@ -0,0 +1,104 @@
+package com.netflix.conductor.aurora;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.conductor.aurora.sql.ResultSetHandler;
+import com.netflix.conductor.core.config.Configuration;
+import com.netflix.conductor.core.utils.PriorityLookup;
+import com.netflix.conductor.dao.ExecutionDAO;
+import com.netflix.conductor.dao.PriorityLookupDAO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.sql.DataSource;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+/**
+ * @author Pradeep Palat
+ */
+
+public class AuroraPriorityLookupDAO extends AuroraBaseDAO implements PriorityLookupDAO {
+
+	private ExecutionDAO executionDAO;
+	private static Logger logger = LoggerFactory.getLogger(AuroraPriorityLookupDAO.class);
+
+	@Inject
+	public AuroraPriorityLookupDAO(DataSource dataSource, ObjectMapper mapper, ExecutionDAO executionDAO, Configuration config) {
+		super(dataSource, mapper);
+		this.executionDAO = executionDAO;
+	}
+
+	public ExecutionDAO getExecutionDAO() {
+		return executionDAO;
+	}
+
+	public void setExecutionDAO(ExecutionDAO executionDAO) {
+		this.executionDAO = executionDAO;
+	}
+
+	public static Logger getLogger() {
+		return logger;
+	}
+
+	public static void setLogger(Logger logger) {
+		AuroraPriorityLookupDAO.logger = logger;
+	}
+
+	@Override
+	public ArrayList<PriorityLookup> getPriority(int priority) {
+		return getWithTransaction( tx->{
+			AuroraPriorityLookupDAO.PriorityLookupHandler handler = new AuroraPriorityLookupDAO.PriorityLookupHandler();
```

Comment:
I would move outside of transaction. That is small cpu time, but anyway. transaction shall be as short as possible

Reply:
Good catch.. Fixed